File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/ProductSpecifications/ProductSpecificationBasic.cs
using Ohtap.ECommerce.Domain.Models.Objects.Specifications;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.ProductSpecifications
{
    public class ProductSpecificationBasic : IProductSpecificationData
    {
        public virtual  string ID { get; set; }
        public virtual int Order { get; set; }
        public virtual string Description { get; set; }
        public virtual int State { get; set; }
        public virtual string Value { get; set; }
        public virtual ISpecificationData Specification { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/ProductSpecifications/ProductSpecificationBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Promotions;





use Ohtap\ECommerce\DomainModels\Objects\Specifications\ISpecificationData;

class ProductSpecificationBasic implements IProductSpecificationData
{

    public string $Description;
    public ISpecificationData $Specification;
    public string $Value;
    public int $State;
    public int $OrderNo;
    public string $Name;
    public string $ID;

}